one-pager/banner/main.py

```
@@ -0,0 +1,71 @@
+from google.oauth2 import service_account
+from google import genai
+from google.genai import types
+from datetime import timedelta
+
+# Replace with your project and bucket details
+PROJECT_ID = "valiant-complex-467519-d3"
+GCS_BUCKET_NAME = "ithaka-one-pagers"
+SERVICE_ACCOUNT_FILE = "key.json"
+
+@functions_framework.http
+def generate_image(request):
```

Ruff (0.12.2)

Missing return type annotation for public function `generate_image`

(ANN201)